addons/website/static/src/scss/website.scss

```diff
@@ -211,6 +211,10 @@ body:not(.o_fullscreen) {
     }
     .nav-item {
         transition: opacity 1000ms ease 0s;
+
+		img.rounded-circle {
+		    object-fit: cover;
+		}
```

Comment:
This rule this looks too generic, and I'm actually not sure to understand what it is trying to target?